user_guide/intro_users_and_benefits.rst

```diff
@@ -0,0 +1,163 @@
+guide to learn more about how OroCRM can be used to analyze information.
+
+
+Communicate and Delegate
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Any CRM user can benefit from OroCRM’s productivity tools which allow any authorized user to:
+
+- Create calendar events for yourself or other users
+
+- Create templates with merge fields or write custom emails in the fly
+
+- Log calls
+
+- Assign tasks to yourself or to others 
+
+- Make notes and comments on different entities within OroCRM
+
+See the :ref:`Productivity Tools section <user-guide-productivity-tools>` of the User Guide to learn more about how OroCRM can help 
+to improve communication and task management within the company.
+
+
+Tools for Sales Representatives
+-------------------------------
+
+Sales representatives have many responsibilities which range from lead generation to closing a sale with a prospective 
+client.  Throughout the process notes are taken and needs are captured.  OroCRM helps sales representatives by providing 
+methods to store this information and by adding structure where needed to simplifying their work with the following 
+capabilities:
+
+
+- 360 degree view of a customer that includes past communication and notes
+
+- Ability to get in touch with customers directly from OroCRM with Google Hangouts or email
+
+- Fully customizable workflows and processes to provide structure to the sales process 
+
+- Tools to collect and process requests received via email or from a website form
+
+Please see the :ref:`Sales Tools section <user-guide-sales-index>` of the User Guide for more details.
+
+  
+Tools for Marketing Associates 
+------------------------------
+
+Marketing associates are responsible for growing a business's customer base.  This is sometimes done through email 
+marketing, customer segmentation, and campaign analysis.  OroCRM facilitates all of these actions and more by providing:
+
+- Segmentation on a specific set of customers by nearly any field  in OroCRM
+
+- Ability to create dynamic targeted marketing lists specific to customer segments
+
+- Ability to send personalized emails using customer data with customizable templates from within OroCRM 
+
+- Automatic synchronization of dynamic marketing lists with 3rd party email marketing platforms such as Mailchimp or 
+  Dotmailer to ensure your contact list is always up to date
+  
+- Tracking of online and offline marketing campaign results in order to study their outcome and effectiveness
+
+See the :ref:`Marketing Tools section <user-guide-marketing>` of the User Guide to learn more about how OroCRM can help 
+with marketing-related activities.
+
+
+
+Tools for Helpdesk and Support 
+------------------------------
+
+Customer service is critical for customer retention in the retail industry.  In fact most consumers say they view 
+customer service as the true test of how much a company values them.  OroCRM can help you maximize customer satisfaction 
+by:
```

Comment:
Edit: In the retail industry, customer service is critical for customer retention. In fact, most consumers say they view customer service as a true test of how much a company values them. OroCRM can help you maximize customer satisfaction by: